### PR TITLE
[Form recognizer] Regenerate for GA version 2023-07-31

### PR DESCRIPTION
**Pull request by samvaity**
Removed:
 - document annotaions
 - queryfields

Added :
 - Abstract class TrainingDataContentSource

### CODE REVIEW

File: sdk/formrecognizer/azure-ai-formrecognizer/src/main/java/com/azure/ai/formrecognizer/documentanalysis/administration/models/AzureBlobFileListContentSource.java [hunk 1]
```diff
@@ -1,13 +1,15 @@
 // Copyright (c) Microsoft Corporation. All rights reserved.
 // Licensed under the MIT License.
+// Code generated by Microsoft (R) AutoRest Code Generator.
```
Comment (samvaity) [inline sdk/formrecognizer/azure-ai-formrecognizer/src/main/java/com/azure/ai/formrecognizer/documentanalysis/administration/models/AzureBlobFileListContentSource.java:3]: remove generation

File: sdk/formrecognizer/azure-ai-formrecognizer/src/main/java/com/azure/ai/formrecognizer/documentanalysis/administration/models/ClassifierDocumentTypeDetails.java [hunk 2]
```diff
@@ -1,63 +1,33 @@
 // Copyright (c) Microsoft Corporation. All rights reserved.
 // Licensed under the MIT License.
+// Code generated by Microsoft (R) AutoRest Code Generator.
```
Comment (samvaity) [inline sdk/formrecognizer/azure-ai-formrecognizer/src/main/java/com/azure/ai/formrecognizer/documentanalysis/administration/models/ClassifierDocumentTypeDetails.java:3]: remove generated

File: sdk/formrecognizer/azure-ai-formrecognizer/src/samples/java/com/azure/ai/formrecognizer/administration/DocumentModelAdminAsyncClientJavaDocCodeSnippets.java [hunk 3]
```diff
@@ -177,10 +179,8 @@ public void beginBuildDocumentClassifierWithOptions() {
         String blobContainerUrl1040D = "{SAS_URL_of_your_container_in_blob_storage}";
         String blobContainerUrl1040A = "{SAS_URL_of_your_container_in_blob_storage}";
         HashMap<String, ClassifierDocumentTypeDetails> docTypes = new HashMap<>();
-        docTypes.put("1040-D", new ClassifierDocumentTypeDetails()
-            .setAzureBlobSource(new AzureBlobContentSource(blobContainerUrl1040D)));
-        docTypes.put("1040-D", new ClassifierDocumentTypeDetails()
-            .setAzureBlobSource(new AzureBlobContentSource(blobContainerUrl1040A)));
+        docTypes.put("1040-D", new ClassifierDocumentTypeDetails(new AzureBlobContentSource(blobContainerUrl1040D)));
+        docTypes.put("1040-D", new ClassifierDocumentTypeDetails(new AzureBlobContentSource(blobContainerUrl1040A)));
```
Comment (samvaity) [inline sdk/formrecognizer/azure-ai-formrecognizer/src/samples/java/com/azure/ai/formrecognizer/administration/DocumentModelAdminAsyncClientJavaDocCodeSnippets.java:183]: 1040-A"

File: sdk/formrecognizer/azure-ai-formrecognizer/src/main/java/com/azure/ai/formrecognizer/documentanalysis/implementation/models/ClassifyDocumentRequest.java [hunk 4]
```diff
@@ -12,13 +12,13 @@
 @Fluent
 public final class ClassifyDocumentRequest {
     /*
-     * Document URL to classify
+     * Document URL to classify.  Either urlSource or base64Source must be specified.
```
Comment (mssfang) [inline sdk/formrecognizer/azure-ai-formrecognizer/src/main/java/com/azure/ai/formrecognizer/documentanalysis/implementation/models/ClassifyDocumentRequest.java:15]: NIT: There are a few places with extra space.

**Review by mssfang — APPROVED** [review]

**Pull request merged**